docs/plugins/testing_workshop_docs/Pythons-assert-keyword.md

```diff
@@ -0,0 +1,91 @@
+# Python's assert keyword
+
+## Article information
+
+This article will be more valuable if you are familiar with the Python programming language and napari software. It is the first in a series of articles on testing taken from the [January 2022 testing workshop video](https://drive.google.com/file/d/1DaMrRz-rLRQ6-_y0J8O3GRpVPCn0rgYs/view). The information in this article starts at minute 5:23. This article is a summary of the information in the video and should stand on its own. The other articles are:
+
+Article 1: This article  
+Article 2: [Pytest testing framework](./Pytest-testing-frameworks)  
+Article 3: [Readers and fixtures](./Readers-and-fixtures)  
+Article 4: [Test coverage](./Test-Coverage)  
```

Comment:
And here as well:

```suggestion
Article 4: [Test coverage](./Test-coverage)  
```

Reply:
Melissa, I am hoping that changing the file names and adding titles to all documents fixed the TOC but I'm not sure.  Please re-review and tell me what you think.